docs/_includes/toc.md

```diff
@@ -0,0 +1,121 @@
+
+<details>
+  <summary><a href="/fprime/INSTALL.html">Install</a></summary>
+    <ul>
+      <li><a href="/fprime/INSTALL.html">Installing F´</a></li>
+      <li><a href="/fprime/UsersGuide/user/autocomplete.html">Installing F´ Console Autocomplete</a></li>
```

Comment:
Can we remove that link to the autocomplete install, and just have the top-level link to the INSTALL.html (no dropdown needed). Also, can we rename that link to `Getting Started`

Reply:
For sure! I might not get to that until next week, but it should be an easy fix.